mlflow/R/mlflow/tests/testthat/test-model.R

```diff
@@ -1,9 +1,11 @@
 context("Model")
 
+library("carrier")
+
 test_that("mlflow can save model function", {
   mlflow_clear_test_dir("model")
   model <- lm(Sepal.Width ~ Sepal.Length, iris)
-  fn <- crate(~ stats::predict(model, .x), model)
+  fn <- crate(~ stats::predict(model, .x), model = model)
```

Comment:
The `carrier` package requires using named arguments to specify data used by the crated function - see https://cran.r-project.org/web/packages/carrier/carrier.pdf and cmd+f for "named arguments"